### PR TITLE
Unlock Divine Authority policy at Transcendend Architecture tech

### LINKED ISSUE #3284: Divine Authority is not unlocked by any tech
Bug Report
==========

* **FreeOrion Version**: 60a10c5

No tech unlocks PLC_DIVINE_AUTHORITY:
```
freeorion$ grep -rl DIVINE default/scripting/*
default/scripting/policies/DIVINE_AUTHORITY.focs.txt
freeorion$
``` 

Maybe, fluff wise, it could be done in TRANSCEND_ARCH. It is not a perfect match, and has no reasoning behind about if that's the right place balance-wise, but I don't see anything better than that.
Alternatively, place it at PSIONICS, but that one is already overloaded of policies (I would relocate some of those, but that's another issue), and you don't need fantastical mind powers to hold divine authority among your people.

### PR DESCRIPTION
**Pull request by Oberlus**
Fixes #3284

### CODE REVIEW

**Comment by geoffthemedio** [comment]
I feel like this should be unlocked much earlier, if not from the start, unless there's some special future spin on it... The concept is historical more so than sci-fi.

**Comment by Oberlus** [comment]
I feel the same. Let's give it some more thought to other techs. But unlocked by start might be fine.

**Comment by hifron** [comment]
There is branch Pre-warp start. No development of DA policy at TA techs? When there were no cars, no proper roads, doesn't mean there were no movement. But to develop cars means not only tech, but something more... Way of life and thinking. And to want something more than develop and travel. But without development there is something like people in Africa. Have techs, but are not developed...

**Comment by geoffthemedio** [comment]
can move later... better to have it unlocked at least somewhere until then

File: default/scripting/techs/construction/TRANSCEND_ARCH.focs.txt [hunk 1]
```diff
@@ -10,7 +10,11 @@ Tech
         "CON_ORGANIC_STRC"
         "CON_NDIM_STRC"
     ]
-    unlock = Item type = Building name = "BLD_MEGALITH"
+    unlock =
```
Comment (geoffthemedio) [inline default/scripting/techs/construction/TRANSCEND_ARCH.focs.txt:13]: Didn't notice earlier or something went weird with merging, but this extra line is a problem...

Comment (Oberlus) [reply default/scripting/techs/construction/TRANSCEND_ARCH.focs.txt:13]: Oops. Yes, that was wrong in my PR, sorry.

**Comment by Oberlus** [comment]
> can move later... better to have it unlocked at least somewhere until then

Yes, it must be moved. Early game, maybe just at start.

Maybe there should be some sort of Social tech unlocked at start, like "Galactic Government/Sociology", with pre-unlocked policies attached to it, and create some, few, new "social" techs to have more space for political/social policies, with two main branches, one for oppression and one for liberty.

**Pull request merged**